HelpSource/Guides/UserFAQ.schelp

```diff
@@ -0,0 +1,418 @@
+
+code::
+SynthDef(\notArray, { |a = (1..5)|
+    a.debug("a is");
+});
+::
+code::a is: an OutputProxy::
+
+code::
+SynthDef(\array, { |a = #[1, 2, 3, 4, 5]|
+   a.debug("a is");
+});
+::
+code::a is: [ an OutputProxy, an OutputProxy, an OutputProxy, an OutputProxy, an OutputProxy ]::
+
+(Note, if 'a' printed as [ 1, 2, 3, 4, 5 ], then you wouldn't be able to change the values in a Synth using code::.set::!)
+
+
+anchor::Server Issues::
+
+SECTION:: Server issues
+
+subsection:: How to trigger a function from the server
+
+The first and most important point: strong::Functions are client-side only::.
+The server doesn't know what functions are, doesn't understand them and has no way to execute them.
+strong::Only the client can execute a function::.
+
+Therefore, if you want a function to execute when something happens in the server, the only way is for the server to tell the client to take the action.
+
+The server can communicate messages back to the client using one of two unit generators: code::SendTrig:: and code::SendReply::.
+code::SendTrig:: is simpler and less flexible (it can send only a code::/tr:: message, and only one data value).
+code::SendReply:: allows you to name the message anything you like, and can send arrays with the message.
+We'll use SendReply here because of its greater flexibility.
+
+Within the language, you also need an object to receive the message and act on it. Usually this is code::OSCresponderNode:: or code::OSCpathResponder::.  In this example, code::OSCpathResponder:: filters messages not just on the name code::/bleep:: but also on the synth's ID. This way, you could have multiple triggering synths, with a different responder and a different action per synth.
+
+code::
+(
+a = {
+    var trig = Dust.kr(8),
+    decay = Decay2.kr(trig, 0.01, 0.1),
+    sig = SinOsc.ar(TExpRand.kr(200, 600, trig), 0, 0.1) * decay;
+    SendReply.kr(trig, '/bleep', trig);
+    sig ! 2
+}.play;
+
+o = OSCpathResponder(s.addr, ['/bleep', a.nodeID], { |time, thisResponder, msg|
+    msg.postln;
+}).add;
+)
+
+a.free; o.remove;
+::
+
+subsection:: Helpfile references:
+
+- link::Classes/SendTrig::, link::Classes/SendReply::
+
+- link::Classes/OSCresponderNode::, link::Classes/OSCpathResponder::, link::Classes/OSCresponder::, link::Guides/OSC_communication::
+
+
+subsection:: Exception in World\_OpenUDP: unable to bind udp socket
+
+Sometime when booting the server one gets a message: code:: Exception in World_OpenUDP: unable to bind udp socket code::.
+
+This is usually caused by an instance of scsynth that as hanged but has not released the osc port, perhaps because it SuperCollider crashed.  To fix this, just hit the strong::"k":: button in the server window to kill all scsynth processes, and then boot again.
```

Comment:
"because it SuperCollider crashed" -> "because SuperCollider crashed"

Also, I'd suggest to replace the sentence starting with "To fix..." with:

> You can use SuperCollider (sclang) to kill all running servers by running `Server.killAll`.
You can also kill scsynth using a terminal or your operating system's task manager.

This is what the server prints nowadays when this error is thrown.